proto/list_hosts.proto

```diff
@@ -34,10 +34,26 @@ message HostList {
         int32 vlan = 4;
 
         // ACLs
-        repeated string acls = 5;
+        repeated ACLData acls = 5;
 
         // link to list_hosts API to list possible destinations
         // or to host_path API to list path
         string url = 6;
     }
-}
\ No newline at end of file
+
+    message ACLData {
```

Comment:
'Data' is bad because *everything* is Data -- so it essentially has no meaning. What is this about an ACL?  Is it just an ACL definition? ACL specification?

Reply:
Using ACLState now, to be consistent with switch_state API.